packages/react-lowcode/src/localization/intl.ts

```diff
@@ -0,0 +1,24 @@
+export function parseMessageId(value: string) : string | undefined {
+    if (value && value.startsWith("intl")) {
```

Comment:
little bit better would be: `if (value?.startsWith('intl.formatMessage('
much better would be to use AST here
please write small jest test for your function

input: `intl.formatMessage('messageId')`
expect: "messageId"

say somehow that this is react-intl (file name, folder, function name ...)